### DISCUSSION TITLE
What discord library should we use?

### DISCUSSION THREAD

**Discussion by nelsonmestevao** [coderdojobraga/gongo#6] [Ideas]
@RuiL1904 told me that `discord.py` is deprecated. A good bet could be [nextcord](https://github.com/nextcord/nextcord) which is a fork with more or less the same API of `discord.py`. 

Any other suggestions?

### References

https://www.reddit.com/r/Discord_Bots/comments/pgma22/best_discordpy_alternative/

**Comment by ruilopesm** [comment]
It seems `discord.py` went back on track on the last month - as we can see from this [gist](https://gist.github.com/Rapptz/c4324f17a80c94776832430007ad40e6). Although `nextcord` is a good choice, staying with `discord.py` seems the best option right now.
